src/styles/components.css

```diff
@@ -49,6 +49,16 @@
     @apply border-red;
   }
 
+  .label {
+    @apply block text-sm after:ml-1
```

Comment:
move `<Label/>` styles to a layer to easily override `text-sm` (overriden by label used in donate page). Once all labels are standardized to be of the new font size - we could move this layer directly to `<Label/>`